tests/test_Restart.cpp

```diff
@@ -258,47 +258,6 @@ std::ostream& operator<<( std::ostream& stream,
     return stream;
 }
 
-bool operator==( const Rates& lhs, const Rates& rhs ) {
-    using rt = Rates::opt;
-
-    BOOST_CHECK_EQUAL( lhs.has( rt::wat ), rhs.has( rt::wat ) );
-    BOOST_CHECK_EQUAL( lhs.has( rt::oil ), rhs.has( rt::oil ) );
-    BOOST_CHECK_EQUAL( lhs.has( rt::gas ), rhs.has( rt::gas ) );
-    BOOST_CHECK_EQUAL( lhs.has( rt::polymer ), rhs.has( rt::polymer ) );
-    BOOST_CHECK_EQUAL( lhs.get( rt::wat, 0.0 ), rhs.get( rt::wat, 0.0 ) );
-    BOOST_CHECK_EQUAL( lhs.get( rt::oil, 0.0 ), rhs.get( rt::oil, 0.0 ) );
-    BOOST_CHECK_EQUAL( lhs.get( rt::gas, 0.0 ), rhs.get( rt::gas, 0.0 ) );
-    BOOST_CHECK_EQUAL( lhs.get( rt::polymer, 0.0 ), rhs.get( rt::polymer, 0.0 ) );
-
```

Comment:
I think you should keep this as is (just changing name of the function). It will help if they are not equal as the error/exception will exactly indicate what is different. The new version will just indicate that something is different.

Reply:
i know of the cost, however, i can't. these are equality operators, not random functions, and they are used deep down in boost test macros.

Reply:
You can, but I agree that it is a lot more manual work. Either using a custom predicate or looping of ther collections yourself and comparing each entry instead of just
```
BOOST_CHECK_EQUAL( lhs.rates, rhs.rates );
```

Reply:
i really do not want to do all that work for a test that is slated for beheading. it's a test with eclCompatibleRST = false which is going away. but if you insist...